src/Microsoft.Azure.Functions.PowerShellWorker.csproj

```diff
@@ -21,11 +21,11 @@ Licensed under the MIT license. See LICENSE file in the project root for full li
 
   <ItemGroup>
     <PackageReference Include="Grpc.Net.Client" Version="2.52.0" />
-    <PackageReference Include="Microsoft.PowerShell.SDK" Version="7.4.0" />
+    <PackageReference Include="Microsoft.PowerShell.SDK" Version="7.4.1" />
```

Comment:
Please upgrade the `.Net 8 SDK` to version `8.0.101`.

Reply:
Done